MapboxCoreNavigation/RouteProgress.swift

```diff
@@ -171,6 +171,16 @@ open class RouteProgress: NSObject {
      */
     public var congestionTimesPerStep: [[[CongestionLevel: TimeInterval]]]  = [[[:]]]
 
+    /**
+     Tuple containing a `SpeedLimit` and a corresponding `CLLocationDistance` representing the position of the SpeedLimit along the current route step
```

Comment:
Nit: Declarations are always present in the generated documentation, so all that’s needed is an explanation of the fields’ semantics.

```suggestion
     A speed limit associated with a distance along a `RouteStep`.
```